wpilibc/src/test/native/cpp/kinematics/DifferentialDriveOdometryTest.cpp

```diff
@@ -57,3 +57,15 @@ TEST(DifferentialDriveWheelSpeeds, GyroAngleReset) {
   EXPECT_NEAR(pose.Translation().Y().to<double>(), 0.0, kEpsilon);
   EXPECT_NEAR(pose.Rotation().Degrees().to<double>(), 0.0, kEpsilon);
 }
+
+TEST(DifferentialDriveOdometry, EncoderDistances) {
+  DifferentialDriveKinematics kinematics{0.381_m * 2};
+  DifferentialDriveOdometry odometry{kinematics, Rotation2d(0_deg)};
```

Comment:
Does this adequately test the tricky gyro offset calculations in the distance version of Update()?